test/integration/render-tests/geojson/fractional-minzoom/style.json

```diff
@@ -0,0 +1,38 @@
+{
```

Comment:
Shoot, I should have thought of this before, but: could we move this test to `render-tests/regressions/mapbox-gl-js#2929`?